ui/imports/shared/views/AssetsView.qml

```diff
@@ -93,7 +93,7 @@ ColumnLayout {
             ExpressionFilter {
                 expression: {
                     d.controller.settingsDirty
-                    return d.tokenIsVisible(model.symbol, model.currentCurrencyBalance) && (customFilter.isCommunity ? !!model.communityId : !model.communityId)
+                    return customFilter.isCommunity ? !!model.communityId : !model.communityId
```

Comment:
This change doesn't make sense here, pls remove

Reply:
It breaks hiding explicitly those selected by the user in `Settings/Manage tokens/Assets/

Reply:
In other works, you have to factor in the following code:
```qml
if (!d.controller.filterAcceptsSymbol(symbol)) // explicitely hidden
  return false
```